docs/source/tutorials/GEDIL4B/GEDI04_L4B documentation.ipynb

```diff
@@ -0,0 +1,293 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    COLLECTIONID = collection['concept-id'] # specify the collection id for the ATLAS dataset
</pre><p>This comment should say "specify the collection id for the GEDI dataset"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/149/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>